### ISSUE TITLE
Subdomain autocalibration

### ISSUE THREAD

**Issue by notsag-dev** [ffuf/ffuf#500]
Hey! Is there any way to do autocalibration for subdomains?

```
ffuf -w  subdomains-top1million-20000.txt -u https://FUZZ.hola.com -r -s -ic -ac 
Error in autocalibration, exiting: Get "https://admingKPJCdzQTkFhPxEr/.hola.com": dial tcp: lookup admingKPJCdzQTkFhPxEr: Temporary failure in name resolution
```
The issue is the slash added after the autocalibration string. Is there a way of not including it? If you need a PR for this I can take a look.

Thanks!

**Comment by 9oelM** [comment]
Happening the same for me. Is your version 1.3.1?

**Comment by 9oelM** [comment]
https://github.com/ffuf/ffuf/issues/384#issuecomment-772599416

I don't think it's supported. I believe it might as well needs to be documented somewhere because I see people often fuzzing across subdomains.

**Comment by joohoi** [comment]
Yeah, subdomain autocalibration is not currently supported. I'm planning to review the calibration functionality in the near future though, trying to figure out some universally working options there. I think the main considerations are:
 - How many autocalibration "extra" requests are needed, and for what kind of tresholds. Doing 5 autocalibration requests for each wordlist entry will grow the amount of traffic five fold naturally.
 - Does the autocalibration support all the use cases, when fuzzing just for subdomains, or multiple subdomains with different paths?
 - Does the autocalibration need to happen on every entry or just initially? ( `ffuf -u https://FUZZ/path` vs. `ffuf -H "Host: FUZZ` )
 - Does the autocalibration need to happen for each URL path? (  `https://example.org/FUZZ` vs. `https://example.org/directory/FUZZ` )
 - What kind of things we want to weed out? Typical universally blocked paths? Just random "every request returns 200" - checks?
 - How to support all of this in an idiomatic, easily understandable way.
 - Finding out the most specific common denominator for the responses.

**Comment by joohoi** [comment]
So the above answer is for the default `-ac` functionality (sounds easier than it is right?). Right now the workaround is to provide your own autocalibration strings with `-acc`.

For example:
```
ffuf -w subdomains.txt -u https://example.org -H "Host: FUZZ.example.org" -acc "knownfalsepositive" -acc "anotherknownfalsepositive" ...
```

**Comment by joohoi** [comment]
This is implemented in the upcoming version v1.5.0 (already available for Kali linux users and sponsors from the exclusive repository)
